db/migrate/20231005071532_rename_columns_in_comment.rb

```diff
@@ -0,0 +1,6 @@
+class RenameColumnsInComment < ActiveRecord::Migration[7.0]
+  def change
+    rename_column :comments, :users_id, :user_id
```

Comment:
- [ ] Please modify this column to be author_id as the foreign key for the users table